src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenReadOnlySpanConstructionTest.cs

```diff
@@ -719,5 +719,190 @@ .maxstack  2
   IL_000b:  ret
 }");
         }
+
+        [Fact]
+        public void StaticFieldIsUsedForSpanCreatedFromArrayWithInitializer_02()
+        {
+            var ilSource = @"
+.class public sequential ansi sealed beforefieldinit System.ReadOnlySpan`1<T>
```

Comment:
Can you leave a comment giving a summary of what this IL is representing? I find it much easier to read IL when I know what it's accomplishing.